### PR TITLE
Added improvements to the input, button and link components after acc…

### PR DESCRIPTION
**Pull request by royderks**
…essibility audit in lighthouse

## Amsterdam styled components pull request

Before opening a pull request, please ensure:

- [ ] The default export from a file matches the file name
- [ ] You have been following the guidelines, written in the [README](../blob/master/docs/CONTRIBUTING.md#user-content-conventions-and-rules) file
- [ ] Your code has the necessary tests written
- [ ] You have updated the [CHANGELOG.md unreleased sections](../blob/master/CHANGELOG.md#muser-content-unreleased)

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

### CODE REVIEW

File: packages/asc-ui/src/components/Button/ShareButton/ShareButton.tsx [hunk 1]
```diff
@@ -6,20 +6,24 @@ import ShareButtonStyle, {
 
 type Props = {
   iconSize?: number
+  title?: string
```
Comment (chrisvanmook) [inline packages/asc-ui/src/components/Button/ShareButton/ShareButton.tsx]: `title` is already part of `HTMLAttributes < HTMLButtonElement >`, so don't need it here

File: packages/asc-ui/src/components/Button/ShareButton/ShareButton.tsx [hunk 2]
```diff
@@ -6,20 +6,24 @@ import ShareButtonStyle, {
...
   iconSize,
+  title,
   ...otherProps
 }) => (
   <ShareButtonStyle
     data-testid="share-button"
     size={30}
     onClick={onClick}
     variant="blank"
+    title={title}
```
Comment (chrisvanmook) [inline packages/asc-ui/src/components/Button/ShareButton/ShareButton.tsx]: `title` here will be spread in otherProps already so not needed here I guess

File: packages/asc-ui/src/components/Menu/MenuToggle/MenuToggle.tsx [hunk 3]
```diff
@@ -10,6 +10,7 @@ const MenuToggle: React.FC<Props> = ({
   onExpand,
   align = 'left',
   open,
+  title,
```
Comment (chrisvanmook) [inline packages/asc-ui/src/components/Menu/MenuToggle/MenuToggle.tsx]: Same here, title will be spread when passed, perhaps merge HTMLAttributes as props as well?

File: packages/asc-ui/src/components/Button/ShareButton/ShareButton.tsx (hunk 2 above)
Comment (royderks) [reply packages/asc-ui/src/components/Button/ShareButton/ShareButton.tsx]: I'm also using `title` for `aria-label`

Comment (chrisvanmook) [reply packages/asc-ui/src/components/Button/ShareButton/ShareButton.tsx]: yes, but aria-label is okay like this, the title prop already exist in HTMLAttributes

File: packages/asc-ui/src/components/Select/Select.tsx [hunk 4]
```diff
@@ -24,37 +24,25 @@ const Select = React.forwardRef<
...
-          </FormLabelStyle>
-        )}
+      <FormLabelStyle
+        htmlFor={id}
+        srOnly={srOnly}
+        label={label}
+        error={errorMessage}
+        style={errorMessage ? errorStyle : labelStyle}
+      >
+        {errorMessage || label}
```
Comment (jonkoops) [inline packages/asc-ui/src/components/Select/Select.tsx]: Probably out of scope for this PR but error messages should really not hijack a label element in order to display an error message. I've created #359 to address this problem.

Comment (royderks) [reply packages/asc-ui/src/components/Select/Select.tsx]: True, thanks!

File: packages/asc-ui/src/components/AmsterdamLogo/AmsterdamLogo.tsx [hunk 5]
```diff
@@ -5,6 +5,7 @@ import { LogoProps } from '../Header/HeaderLogoText'
 const AmsterdamLogo: React.FC<LogoProps> = ({ ...props }) => (
   <AmsterdamLogoStyle {...props}>
     <LogoStyle />
+    Gemeente Amsterdam
```
[the remark this answers is not in the document]
Comment (royderks) [reply packages/asc-ui/src/components/AmsterdamLogo/AmsterdamLogo.tsx]: Didn't know this one, will update :)

**Pull request merged**